index.html

```diff
@@ -139,36 +138,36 @@ <h2 class="portfolio-text">Portfolio</h2>
             </div>
 
             <div class="portfolio-boxes">
-                <img src="img/portfolio/portfolio-1.svg" alt="photo 1" class="portfolio-photo">
-                <img src="img/portfolio/portfolio-2.svg" alt="photo 2" class="portfolio-photo">
-                <img src="img/portfolio/portfolio-3.svg" alt="photo 3" class="portfolio-photo">
-                <img src="img/portfolio/portfolio-4.svg" alt="photo 4" class="portfolio-photo">
-                <img src="img/portfolio/portfolio-5.svg" alt="photo 5" class="portfolio-photo">
-                <img src="img/portfolio/portfolio-6.svg" alt="photo 6" class="portfolio-photo">
-                <img src="img/portfolio/portfolio-7.svg" alt="photo 7" class="portfolio-photo">
-                <img src="img/portfolio/portfolio-8.svg" alt="photo 8" class="portfolio-photo">
-                <img src="img/portfolio/portfolio-9.svg" alt="photo 9" class="portfolio-photo">
-                <img src="img/portfolio/portfolio-10.svg" alt="photo 10" class="portfolio-photo">
-                <img src="img/portfolio/portfolio-11.svg" alt="photo 11" class="portfolio-photo">
-                <img src="img/portfolio/portfolio-12.svg" alt="photo 12" class="portfolio-photo">
-                <img src="img/portfolio/portfolio-1.svg" alt="photo 1" class="portfolio-photo">
+                <img src="/img/portfolio/portfolio-1.svg" alt="photo 1" class="portfolio-photo">
```

Comment:
Can you please make all images to start from ./img/....
It's a goot practice to implement how you did, but you should provide in this case description in README.md file how to run your application in a way to see all imeges.